_data/governments.yml

```diff
@@ -95,6 +95,7 @@ Brazil:
   - tisorocaba
   - tre-pa
   - wpgovbr
+  - EscolaDeSaudePublica
```

Comment:
CI to check the alphabetize just landed, so you probably need to pull this up in the list